rootfs/api/tests/test_certificate.py

```diff
@@ -120,3 +121,28 @@ def test_delete_certificate(self):
         url = '/v2/certs/random-test-cert'
         response = self.client.delete(url)
         self.assertEqual(response.status_code, 204)
+
+    def test_create_invalid_cert(self):
+        """Upload a cert that can't be loaded by pyopenssl"""
+        response = self.client.post(
+            self.url,
+            {
+                'name': 'random-test-cert',
+                'certificate': 'i am bad data',
+                'key': 'i am bad data as well'
+            }
+        )
+        self.assertEqual(response.status_code, 400)
+        # match partial since right now the rest is pyopenssl errors
```

Comment:
We wrap the exception in `ValidationError` and SupsciousOperation` depending on which part of the system it happens in so we couldn't catch `pyoenssl.Exception`, right now anyway :|